lib/workers/global/config/parse/file.ts

```diff
@@ -90,6 +91,14 @@ export async function getConfig(env: NodeJS.ProcessEnv): Promise<AllConfig> {
     );
     config = migratedConfig;
   }
+
+  const { warnings, errors } = await validateConfig(migratedConfig);
+  if (warnings.length) {
+    logger.warn({ warnings }, 'Found warnings in configuration');
```

Comment:
Should I use the name of the configuration here? ie. file config or bot config.
It might confuse users if we don't mention which config the errors are related to.